hack/e2e.sh

```diff
@@ -4,9 +4,37 @@ set -euo pipefail
 
 GOPATH="$(mktemp -d)"
 export GOPATH
+AUTOSCALER_PKG="k8s.io/autoscaler"
```

Comment:
I would prefer it if we could have this test aim at a specific fork of the `openshift/kubernetes-autoscaler` repo. That way we can have OpenShift-specific patches if we need, and by using a specific fork, older branches of the CI will target older branches of the VPA. We want to avoid having broken CI on an old release branch when in the future, it tries to use the latest VPA, which might have a test for a new feature which the old release branch doesn't support.

If you would like, I can work on some makefile code to handle this case.